hip-config.cmake.in

```diff
@@ -86,8 +86,10 @@ if(HIP_COMPILER STREQUAL "clang")
           ${HIP_CLANG_INCLUDE_SEARCH_PATHS}
       NO_DEFAULT_PATH)
   find_dependency(AMDDeviceLibs)
-  set(AMDGPU_TARGETS "gfx900;gfx906;gfx908" CACHE STRING "AMD GPU targets to compile for")
```

Comment:
This variable can't be removed as there are many libraries still using this variable. Instead, there should be a warning if the `AMDGPU_TARGETS` is set. This will give some time for the libraries to update the variable names, and then at a later time this variable can be removed.

Reply:
@pfultz2 Do you know where I can find the `hip-config.cmake.in` in the new develop branch? 
I like to create new PR because of the strange behavior of two variables where one can not be changed with `ccmake` still exists in HIP 4.3 but it looks like the cmake stuff is maybe shifted into a new repository during the introduction of the new `develop` branch.

Reply:
OK, I found the new repo https://github.com/ROCm-Developer-Tools/hipamd